src/site-packages/openbmp/mrt2bmp/MrtProcessors.py

```diff
@@ -0,0 +1,753 @@
+
+         Type = 0: String
+         Type = 1: reason
+
+        """
+
+        # Creation of Type 1 reason tlv
+        information = struct.pack("!H", 0)
+        reason_tlv = struct.pack("!H H", 1, len(information)) + information
+
+        common_header = self.createBmpCommonHeader(3, len(reason_tlv) + 6, 5)
+
+        qm = common_header + reason_tlv
+
+        return qm
+
+    def processRouteView(self):
+
+        if self._isToProcess:
+            for f in self._listOfRibAndUpdateFiles:
+
+                print "-- %s is started" % f[1]
+
+                if "rib" in f[1]:
+
+                    rp = RibProcessor(f[1], self._directory_path, self._router_name, self._collector_id, self._fwd_queue, self._log_queue)
+
+                    rp.processRibFile()
+
+                    moveFileToTempDirectory(os.path.join(self._directory_path, self._router_name, f[1]),
+                                            os.path.join(self._processed_directory_path, self._router_name, "RIBS"))
+
+                elif "updates" in f[1]:
+
+                    up = UpdateProcessor(f[1], self._directory_path, self._router_name, self._collector_id, self._fwd_queue, self._log_queue)
+                    up.processUpdateFile()
+
+                    moveFileToTempDirectory(os.path.join(self._directory_path, self._router_name, f[1]),
+                                            os.path.join(self._processed_directory_path, self._router_name, "UPDATES"))
+
+                print "-- %s is ended" % f[1]
+
+        else:
+            self.LOG.error("Data of %s cannot be processed..." % self._router_name)
+
+class RouteViewsProcessor(multiprocessing.Process):
+
+    def __init__(self, cfg, log_queue, fwd_queue):
+        """ Constructor
+
+            :param cfg:             Configuration dictionary
+            :param forward_queue:   Output for BMP raw message forwarding
+            :param log_queue:       Logging queue - sync logging
+        """
+
+        multiprocessing.Process.__init__(self)
+        self._stop = multiprocessing.Event()
+
+        self.cfg = cfg
+        self._cfg_router = cfg['router_data']
+        self._cfg_collector = cfg['collector']
+        self._log_queue = log_queue
+        self._router_dir_path = self._cfg_router['router_directory_path']
+        self.LOG = None
+
+        # Start the BMP writer process
+        self._fwd_queue = fwd_queue
+        self._collector_writer = None
+
+    def run(self):
+        """ Override """
+        self.LOG = init_mp_logger("mrt_processors", self._log_queue)
+
+        self.LOG.info("Running route_views_processor")
+
+        # Parse name of router from router directory name.
+        if os.path.isdir(self._router_dir_path):
+            self._dir_path, self.router_name = os.path.split(self._router_dir_path)
+
+        else:
+            print "%s is not a directory !" % self._router_dir_path
+            self.stop()
+
+        try:
+            print "- %s is started" % str(self.router_name)
```

Comment:
Use logger (e.g. LOG.info() ) instead of print.  Logger will correctly timestamp the messages to the configured appender.